warehouse/internal/loadfiles/loadfiles.go

```diff
@@ -97,11 +104,14 @@ type WorkerJobRequest struct {
 	StagingUseRudderStorage      bool
 	UniqueLoadGenID              string
 	RudderStoragePrefix          string
-	Output                       []WorkerJobResponse
 	LoadFilePrefix               string // prefix for the load file name
 	LoadFileType                 string
 }
 
+type WorkerJobRequestMetadata struct {
```

Comment:
Can't we just pass the upload schema instead? What other metadata are we planning to add?